x/wasm/keeper/legacy_querier_test.go

```diff
@@ -293,7 +291,7 @@ func TestLegacyQueryContractHistory(t *testing.T) {
 			var defaultQueryGasLimit sdk.Gas = 3000000
 			q := NewLegacyQuerier(keeper, defaultQueryGasLimit)
 			queryContractAddr := spec.srcQueryAddr
-			if queryContractAddr.Empty() {
+			if queryContractAddr == nil {
```

Comment:
This is changed from `CosmWasm/wasmd-v0.27.0`. We should be the same since we continue to merge from `CosmWasm/wasmd`.
* https://github.com/CosmWasm/wasmd/blob/v0.27.0/x/wasm/keeper/legacy_querier_test.go#L294